nucleus/src/main/java/za/co/absa/subatomic/infrastructure/team/TeamAutomationHandler.java

```diff
@@ -227,18 +224,77 @@ public void teamMembersAdded(TeamEntity teamEntity,
         }
     }
 
+    public void teamMemberRemoved(TeamEntity teamEntity,
+                                  TeamMemberEntity memberEntity,
+                                  TeamMemberEntity requesterEntity) {
+
+        za.co.absa.subatomic.domain.team.SlackIdentity teamEntitySlackIdentity = null;
+
+        if (teamEntity.getSlackDetails() != null) {
+            teamEntitySlackIdentity = new za.co.absa.subatomic.domain.team.SlackIdentity(
+                    teamEntity.getSlackDetails()
+                            .getTeamChannel());
+        }
+
+        Team team = new Team(teamEntity.getTeamId(), teamEntity.getName(),
+                teamEntitySlackIdentity);
+
+        TeamMember memberRemoved = new TeamMember(
+                memberEntity.getMemberId(),
+                memberEntity.getDomainUsername(),
+                memberEntity.getFirstName(),
+                memberEntity.getLastName(),
+                memberEntity.getEmail(),
+                GetTeamMemberSlackIdentity(memberEntity));
+
+        TeamMember memberRequested = new TeamMember(
+                requesterEntity.getMemberId(),
+                requesterEntity.getDomainUsername(),
+                requesterEntity.getFirstName(),
+                requesterEntity.getLastName(),
+                requesterEntity.getEmail(),
+                GetTeamMemberSlackIdentity(requesterEntity));
+
+        // Build the ingestable object
+        MemberRemovedFromTeam ingestableObject = new MemberRemovedFromTeam(
+                team, memberRemoved, memberRequested);
+
+        log.info("A member has been removed from a team, sending event to Atomist for ingestion", ingestableObject);
+
+        ResponseEntity<String> response = restTemplate.postForEntity(
+                atomistConfigurationProperties
+                        .getMembersRemovedFromTeamEventUrl(),
+                ingestableObject,
+                String.class);
+
+        if (response.getStatusCode().is2xxSuccessful()) {
+            log.info("Atomist has ingested event memberRemovedEvent successfully: {} -> {}",
+                    response.getHeaders(), response.getBody());
+        }
+    }
+
+    private TeamMemberSlackIdentity GetTeamMemberSlackIdentity(TeamMemberEntity memberEntity)
```

Comment:
This is C# naming you are using here :) java needs the first letter of a function to be lowercase.